packages/taquito-ledger-signer/test/taquito-ledger-signer.spec.ts

```diff
@@ -185,17 +185,24 @@ describe('LedgerSigner test', () => {
   });
 
   it('Should sign Operation for tz1 bip32', async () => {
-    const signer = new LedgerSigner(mockTransport, "44'/1729'/1'/0'", true, DerivationType.BIP32_ED25519);
+    const signer = new LedgerSigner(
+      mockTransport,
+      "44'/1729'/1'/0'",
+      true,
+      DerivationType.BIP32_ED25519
+    );
```

Comment:
was this from a merge conflict? 
if this is different from master it could not be working

Reply:
I think this is also a spacing/formatting automated change from lint